classes/CMSCategory.php

```diff
@@ -496,28 +577,34 @@ public function getLink(Link $link = null)
         return $link->getCMSCategoryLink($this->id, $this->link_rewrite);
     }
 
-    public function getName($id_lang = null)
+    /**
+     * @param null $idLang
+     *
+     * @return string
+     */
+    public function getName($idLang = null)
     {
         $context = Context::getContext();
-        if (!$id_lang) {
+        if (!$idLang) {
```

Comment:
not realy, just want to make it very explicit: for instance,

```php
if (!is_null($idLang)) { }
```
allows one and only one available value (null) to enter in the if condition.

Or maybe I didn't understood the previous condition, what do you think ?